test/TestAsset/WhereInvokable.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace ZendTest\Db\TestAsset;
+
+use Zend\Db\Sql\Predicate\Like;
+use Zend\Db\Sql\Select;
+
+class WhereInvokable
+{
+    private $value;
+
+    /**
+     * WhereInvokable constructor.
+     * @param $value
+     */
+    public function __construct($value)
```

Comment:
Add a type declaration to `$value`

Reply:
I did not add the type declaration because the `composer.json` says this module is still on php 5.5. If this is an outdated information, I'll add the type.

Reply:
Hmm, weird that it is still on 5.5. `develop` should be on `7.1`

Reply:
yeah, also travis runs with an old configuration

Reply:
@I3ekka we are going to support PHP 5.6+ with zend-db 2.9.0. We'll move to PHP 7.1 with zend-db 3.0.0